src/hsd/common.py

```diff
@@ -26,18 +31,21 @@ def unquote(txt):
 # Suffix to mark attribute
 ATTRIB_SUFFIX = ".attrib"
 
-# Length of the attribute suffix
-LEN_ATTRIB_SUFFIX = len(ATTRIB_SUFFIX)
-
 # Suffix to mark hsd processing attributes
 HSD_ATTRIB_SUFFIX = ".hsdattrib"
 
-# Lengths of hsd processing attribute suffix
-LEN_HSD_ATTRIB_SUFFIX = len(HSD_ATTRIB_SUFFIX)
-
+# HSD attribute containing the original tag name
+HSD_ATTRIB_NAME = "name"
 
+# HSD attribute containing the line number
 HSD_ATTRIB_LINE = "line"
 
+# HSD attribute marking that a node equals to its only child (instead of
```

Comment:
```suggestion
# HSD attribute marking that a node is equal to its only child (instead of
```
In this case, no pluralisation (whatever equality means for general structures).